spyderlib/config.py

```diff
@@ -530,6 +557,7 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
+              'editor/save as': 'Ctrl+Alt+S',

```

Comment:
I agree with @Nodd, every other app uses `Ctrl+Shift+S` for `Save As`.  We should move Save All to `Ctrl+Alt+S`